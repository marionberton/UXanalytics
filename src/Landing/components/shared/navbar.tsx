import React from 'react';
import { Link } from 'react-router-dom';

import logo from '../../../assets/logo.svg';

export const Navbar: React.FC = () => {
  return (
    <nav className=" container pt-16 flex items-center justify-between flex-wrap">
      <div className="flex items-center flex-shrink-0 text-white mr-6">
        <img src={logo} alt="logo" />
      </div>
      <div>
        <Link
          to="/signin"
          className="inline-block  bg-white text-blue-800 font-title font-semibold py-2 px-4 rounded-full"
        >
          Try Demo Now
        </Link>
      </div>
    </nav>
  );
};
