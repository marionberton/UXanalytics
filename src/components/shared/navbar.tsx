import React from 'react';

import logo from '../../assets/logo.svg';

export const Navbar: React.FC = () => {
  return (
    <>
      <nav className=" container  mt-16 flex items-center justify-between flex-wrap">
        <div className="flex items-center flex-shrink-0 text-white mr-6">
          <img src={logo} alt="logo" />
        </div>
        <div>
          <a
            href="#"
            className="inline-block  bg-white text-blue-800 font-title font-semibold py-2 px-4 rounded-full"
          >
            Try Demo Now
          </a>
        </div>
      </nav>
    </>
  );
};
