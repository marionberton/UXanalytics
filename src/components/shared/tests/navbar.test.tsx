import React from 'react';
import { mount } from 'enzyme';

import { Navbar } from '../navbar';

describe('<Navbar />', () => {
  it('renders the component', () => {
    const component = mount(<Navbar />);

    expect(component.render()).toMatchSnapshot();
  });
});
