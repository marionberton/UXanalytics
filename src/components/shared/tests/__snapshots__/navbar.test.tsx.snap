// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> renders the component 1`] = `
<nav
  class=" container mt-16 flex items-center justify-between flex-wrap"
>
  <div
    class="flex items-center flex-shrink-0 text-white mr-6"
  >
    <img
      alt="logo"
      src="logo.svg"
    />
  </div>
  <div>
    <a
      class="inline-block  bg-white text-blue-800 font-title font-semibold py-2 px-4 rounded-full"
      href="/signin"
    >
      Try Demo Now
    </a>
  </div>
</nav>
`;
